predicate/predicate.py

```diff
@@ -1,53 +1,17 @@
 import re
 
+from django.db.models.manager import Manager
 from django.db.models.query_utils import Q
+from django.utils.datastructures import SortedDict
```

Comment:
Are we supporting Django 1.7 here? If so I'll need to a `try/except` import block to support both structures.